src/mlpack/methods/lars/lars_impl.hpp

```diff
@@ -794,9 +772,11 @@ LARS<ModelMatType>::Train(const MatType& matX,
         // need to take a step with the previous beta direction towards the next
         // variable we will add.
         s = s.subvec(0, activeSet.size() - 1); // Drop last element.
+        matGramActive = matGramActive.submat(0, 0, activeSet.size() - 1,
+            activeSet.size() - 1);
         matS = s * ones<MatType>(1, activeSet.size());
         // This worked last iteration, so there can't be a singularity.
-        solve(unnormalizedBetaDirection,
+        const bool status2 = solve(unnormalizedBetaDirection,
```

Comment:
@rcurtin `status2` doesn't seem to be used anywhere

Reply:
Thanks!  Removed in c8d40be8.